sssd_test_framework/roles/ipa.py

```diff
@@ -811,6 +1040,151 @@ def __get_member_args(self, members: list[IPAUser | IPAGroup | str]) -> list[str
         external = [x for item in members if isinstance(item, str) for x in ("--external", item)]
         return [*users, *groups, *external]
 
+    def idgroupoverride(self) -> IDGroupOverride:
+        """
+        Add override to the IPA Group.
+
+        .. code-block:: python
+            :caption: Example usage
+
+            @pytest.mark.topology(KnownTopology.IPA)
+            def test_example(client: Client, ipa: IPA):
+                ipa.idview("newview1").add(description="This is a new view")
+                ipa.idview("newview1").apply(hosts=f"{client.host.hostname}")
+                ipa.group("group-1").add().idgroupoverride().add_override("newview1", gid=1344567)
+                client.sssd.restart()
+                g_lookup = client.tools.getent.group("group-1")
+                assert g_lookup.gid == 1344567
+
+        :return: New IDOverride object.
+        :rtype: IDOverride
+        """
+        return IDGroupOverride(self)
+
+
+class IDGroupOverride(IPAGroup):
+    """
+    IPA group ID override
+    """
+
+    def __init__(self, group: IPAGroup) -> None:
+        """
+        :param user: IPA group object
+        :type user: IPAGroup
+        """
+        super().__init__(group.role, group.name)
+        self.name = group.name
+
+    def add_override(
+        self,
+        idview_name: str,
+        *,
+        description: str | None = None,
+        name: str | None = None,
+        gid: int | None = None,
+        **kwargs,
```

Comment:
What is the purpose of `*` and `**kwargs` if all possible arguments are named?

Reply:
Definitely, I can remove `**kwargs` but,
With * (current behavior):

- idview_name is a positional argument.

- description, name, and gid are keyword-only arguments (must be specified explicitly like add_override("view1", name="test")).

Without *:

- All arguments, including description, name, and gid, can be passed positionally or as keywords (add_override("view1", "desc", "test", 1001)).

I am also okay to remove `⁣*` but I think good to have it.
Please, let me know if I am mistaken.

Reply:
It was added while testing parameterization,
https://github.com/SSSD/sssd/pull/7888/files#diff-1b413e9272404115544de5e1b2d573853586d9edf35bd0bd80f61b51077c081aR476
Yes, it was added only in `add_override()` for group.
Also thinking to have same for non-valid attribute for group,
https://github.com/SSSD/sssd-test-framework/pull/154/files#diff-a97a14df2c8e3f1d6e9c0878a73fed25bbbe15074d0ad17f47f82d4f14b4f259R764

Reply:
@alexey-tikhonov can I resolve this?

Reply:
ok